src/lib/storage/storage.ts

```diff
@@ -289,6 +298,31 @@ export interface CredentialMetadataSummary extends CredentialMetadata {
   issuer: IdentitySummary
 }
 
+const cacheDIDDoc = (connection: Connection) => (doc: DidDocument) => {
+  const cacheEntry = plainToClass(CacheEntity, {
+    // TODO Change to did?
```

Comment:
I think we discussed that, so this TODO can be removed (also in the get method)

Reply:
Can do.